hack/dev/access-entries-username-prefix.template

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Is this a test fixture? If so, can we move it into a subdirectory with the name `testfixture` or something?

Reply:
yep, currently hack/dev/ is the directory holding all the test fixture. Would you like to move them to a subdir under hack/dev/

Reply:
ah so this is just setup infra. After this PR, we should move all the Role creation/configuration outside of this repo and make sure its just set up and ready to use